sdk/python/src/dagger/client/gen.py

```diff
@@ -436,7 +446,8 @@ def as_tarball(
         _args = [
             Arg(
                 "platformVariants",
-                [] if platform_variants is None else platform_variants,
+                () if platform_variants is None else platform_variants,
+                (),
```

Comment:
Got confused because I remembered fixing this! Took me a while to realize I fixed this **bug** in this PR 😂